### PR TITLE
Basic setup for the files/bundles/their unit tests to communicate with GCS.

### PR DESCRIPTION
**Pull request by ttung**
Unit test in the next diff.

### CODE REVIEW

File: dss/config.py [hunk 1]
```diff
@@ -4,23 +4,37 @@
...
 
 class Config(object):
     _S3_BUCKET = None  # type: str
+    _GS_BUCKET = None  # type: str
 
     @staticmethod
     def get_cloud_specific_handles(replica: str) -> typing.Tuple[
             BlobStore, HCABlobStore, str]:
+
+        handle: BlobStore
```
Comment (kislyuk) [inline dss/config.py]: Hmm... I don't understand this syntax, what is this type hint attached to?

Comment (ttung) [inline dss/config.py]: handle.

type hinting is currently broken in python if the same variable is declared in two different paths of an if-statement.

File: dss/config.py [hunk 2]
```diff
@@ -4,23 +4,37 @@
...
         if replica == 'aws':
             handle = S3BlobStore()
             return (
                 handle,
                 S3HCABlobStore(handle),
                 Config.get_s3_bucket()
             )
+        elif replica == 'gcs':
+            pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+            credentials = os.path.join(pkg_root, "gcs-credentials.json")
```
Comment (kislyuk) [inline dss/config.py]: I think we want to always use the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, no?

Comment (ttung) [inline dss/config.py]: This is wrong in multiple places.  I'll fix in a follow-up PR.

**Pull request merged**